APCAppCore/APCAppCore/Library/MedicationTrackingAppComponent/APCMedicationTrackerDetailViewController.m

```diff
@@ -120,7 +120,9 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
     
     cell.medicationName.text = [self extractMedicationNamePrefix:self.lozenge.prescription.medication.name];
     
-    NSString  *doseNumberString = [NSString stringWithFormat:@"Dose %ld", (indexPath.row + 1)];
+    NSString *doseFormat = NSLocalizedStringWithDefaultValue(@"Dose Quantity", @"APCAppCore", APCBundle(), @"Dose %@",
+                                                             @"Title for a list item with a medical dose of %@ amount.");
+    NSString  *doseNumberString = [NSString stringWithFormat:doseFormat, @(indexPath.row + 1)];
```

Comment:
There's a macro I defined in APHParkinsonActivityViewController.h, APHLocalizedStringFromNumber(), that maybe should be moved into APCBaseTaskViewController.h (or even APCLocalization.h) and renamed APCLocalized...etc? (would require updating mPower as well) anyway, either that, or similar logic should be followed here to localize the number format. i.e. [NSNumberFormatter localizedStringFromNumber:@(indexPath.row + 1) numberStyle:NSNumberFormatterNoStyle] in place of @(indexPath.row + 1)